*** useTransceiverCtrl.ts	2025-08-20 08:59:51.115108479 +0900
--- useTransceiverCtrl.ts.org	2025-08-20 13:28:47.731265485 +0900
***************
*** 561,570 ****
      // }
  
      // 人工衛星がドップラーシフトが有効となる範囲にいるか判定する
!     // const isActive = await isWithinDopplerShiftActiveRange();
!     // if (!isActive) {
!     //   return;
!     // }
  
      // 無線機からの周波数データ(トランシーブ)受信があった場合はドップラーシフトを待機する
      ApiTransceiver.dopplerShiftWaitingCallback(async (res: ApiResponse<boolean>) => {
--- 561,570 ----
      // }
  
      // 人工衛星がドップラーシフトが有効となる範囲にいるか判定する
!     const isActive = await isWithinDopplerShiftActiveRange();
!     if (!isActive) {
!       return;
!     }
  
      // 無線機からの周波数データ(トランシーブ)受信があった場合はドップラーシフトを待機する
      ApiTransceiver.dopplerShiftWaitingCallback(async (res: ApiResponse<boolean>) => {
***************
*** 587,608 ****
                currentDate.value,
                autoTrackingIntervalMsec
              );
!             // 変更後のTx周波数でドップラーシフトの基準周波数を更新したい、が変更前の値が返る
!             // const tempTxFrequency = TransceiverUtil.parseNumber(txFrequency.value);
!             // dopplerTxBaseFrequency.value = tempTxFrequency / txDopplerFactor;
  
              // Rxドップラーファクターを計算する
              const rxDopplerFactor = await frequencyTrackService.calcDownlinkDopplerFactor(
                currentDate.value,
                autoTrackingIntervalMsec
              );
!             // 変更後のRx周波数でドップラーシフトの基準周波数(Rx)を更新する
              const tempRxFrequency = TransceiverUtil.parseNumber(rxFrequency.value);
!             dopplerRxBaseFrequency.value = tempRxFrequency / rxDopplerFactor;
! 
!             // ドップラー基準周波数(Tx)を計算する:定数はRS-44の値
!             dopplerTxBaseFrequency.value = 581602600 - dopplerRxBaseFrequency.value;
!             const tempTxFrequency = dopplerTxBaseFrequency.value * txDopplerFactor;
            }
          }, Constant.Transceiver.TRANSCEIVE_WAIT_MS);
          return;
--- 587,610 ----
                currentDate.value,
                autoTrackingIntervalMsec
              );
!             // 変更後のTx周波数でドップラーシフトの基準周波数を更新する
!             const tempTxFrequency = TransceiverUtil.parseNumber(txFrequency.value);
!             dopplerTxBaseFrequency.value = TransceiverUtil.subtractFrequencies(
!               tempTxFrequency,
!               tempTxFrequency * (1 - txDopplerFactor)
!             );
  
              // Rxドップラーファクターを計算する
              const rxDopplerFactor = await frequencyTrackService.calcDownlinkDopplerFactor(
                currentDate.value,
                autoTrackingIntervalMsec
              );
!             // 変更後のRx周波数でドップラーシフトの基準周波数を更新する
              const tempRxFrequency = TransceiverUtil.parseNumber(rxFrequency.value);
!             dopplerRxBaseFrequency.value = TransceiverUtil.subtractFrequencies(
!               tempRxFrequency,
!               tempRxFrequency * (1 - rxDopplerFactor)
!             );
            }
          }, Constant.Transceiver.TRANSCEIVE_WAIT_MS);
          return;
